### PR TITLE
EF hacktoberfest

### PR DESCRIPTION
**Pull request by marehead**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-uk.md [hunk 1]
```diff
@@ -19,19 +18,12 @@
...
 
-* [Вивчить собі Хаскела на велике щастя!](http://haskell.trygub.com) - Міран Ліповача
-
-
-### HTML / CSS
-
-#### Bootstrap
-
-* [Ознайомлення Bootstrap 3.3.2](http://twbs.docs.org.ua)
+* [Вивчити собі Хаскель на велике щастя!](http://haskell.trygub.com) - Міран Ліповача
```
Comment (StepanNaryshkov) [inline books/free-programming-books-uk.md:21]: `Вивчить собі Хаскела на велике щастя!`
Why did you change the name? It was correct before

image

File: books/free-programming-books-uk.md [hunk 2]
```diff
@@ -19,19 +18,12 @@
...
 
 
 ### Java
 
-* [Програмування мовою Java для дітей, батьків, дідусів та бабусь](http://myflex.org/books/java4kids/java4kids.htm) - Яків Файн
+* [Програмування на Java для дітей, батьків, дідусів та бабусь](http://myflex.org/books/java4kids/java4kids.htm) - Яків Файн
```
Comment (StepanNaryshkov) [inline books/free-programming-books-uk.md:26]: Why did you change the original name? 
image

File: books/free-programming-books-uk.md [hunk 3]
```diff
@@ -44,6 +36,12 @@
 * [Дизайн патерни - просто, як двері](http://designpatterns.andriybuday.com) - А. Будай
 
 
+### PHP
+
+* [Symfony. Швидкий старт](https://symfony.com/doc/current/the-fast-track/uk/index.html) - Symfony SAS
+* [Документація Laravel 8.x](https://www.docs-laravel.site/docs/8.x/) - Laravel.su
```
Comment (StepanNaryshkov) [inline books/free-programming-books-uk.md:42]: this link does not work, 
image

**Comment by shrn01** [comment]
Looks like u have taken a wrong branch. Please take `main` and update your changes on top of it and update the PR

**Comment by eshellman** [comment]
I think you're going to want to start from a fresh branch
